Packs/Slack/pack_metadata.json

```diff
@@ -32,9 +32,5 @@
             "mandatory": false,
             "display_name": "Active Directory Query"
         }
-    },
-    "marketplaces": [
-        "xsoar",
-        "marketplacev2"
```

Comment:
why remove?

Reply:
Oy. When I merged from master, I said to take my changes. Wasn't aware there were actual changes lol. I'll put it back